nautobot/core/tests/test_authentication.py

```diff
@@ -125,21 +123,20 @@ def test_remote_auth_auto_create(self):
     @override_settings(
         AUTHENTICATION_BACKENDS=TEST_AUTHENTICATION_BACKENDS,
         REMOTE_AUTH_AUTO_CREATE_USER=True,
-        REMOTE_AUTH_DEFAULT_GROUPS=["Group 1", "Group 2"],
-        REMOTE_AUTH_ENABLED=True,
+        EXTERNAL_AUTH_DEFAULT_GROUPS=["Group 1", "Group 2"],
     )
-    def test_remote_auth_default_groups(self):
+    def test_EXTERNAL_AUTH_DEFAULT_groups(self):
```

Comment:
Not sure I'm a fan of the mixed-case rename of this test case and the following one. Would you mind changing them to standard lower-case?